airflow/config_templates/default_airflow.cfg

```diff
@@ -488,6 +491,16 @@ bind_password = insecure
 basedn = dc=example,dc=com
 cacert = /etc/ca/ldap_ca.crt
 search_scope = LEVEL
+basedn_group= dc=example,dc=com
```

Comment:
Cette variable pourrait être groupée avec le autres ?

Reply:
Yeah ! It's done